qCC/db_tree/sfEditDlg.cpp

```diff
@@ -44,19 +44,15 @@ sfEditDlg::sfEditDlg(QWidget* parent/*=0*/)
 
 	//histogram window
 	{
-		m_associatedSFHisto = new ccHistogramWindow();
+		m_associatedSFHisto = new ccHistogramWindow;
 		QHBoxLayout* hboxLayout = new QHBoxLayout(m_ui->histoFrame);
 		hboxLayout->addWidget(m_associatedSFHisto);
 		hboxLayout->setContentsMargins(0, 0, 0, 0);
 		m_associatedSFHisto->enableSFInteractionMode(true);
 		m_associatedSFHisto->xAxis->setTickLabels(false);
-		//m_associatedSFHisto->xAxis->setAutoSubTicks(false);
-		//m_associatedSFHisto->xAxis->setSubTickCount(0);
-		m_associatedSFHisto->xAxis->setAutoTickCount(6);
+		m_associatedSFHisto->xAxis->ticker()->setTickCount(6);
```

Comment:
I was also not 100% on these conversions from `setAutoTickCount`. They moved from setting it directly to a ticker class that doesn't quite have the same things (as far as I can tell).

Reply:
Same thing, it seems to work fine isn't it?